### ISSUE TITLE
Split INIUpdater to GUI/non-GUI parts

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#4465]
e.g. use CMake defined macros or create INIUpdater and TOPPASUpdater.
See #4462

**Comment by bernt-matthias** [comment]
Maybe as well for the ImageCreator..

**Comment by jpfeuffer** [comment]
Yes would be cool, but maybe harder. I think I looked at it, and actually the support for writing svg and other image formats comes from a Qt GUI dependency I think. One would need to recheck.

**Comment by bernt-matthias** [comment]
`Resampler` seems to be another tool that is missing from the conda distribution for this reason.

**Comment by timosachsenberg** [comment]
done
